src/main/java/org/folio/helper/BindHelper.java

```diff
@@ -88,34 +90,41 @@ private Future<Map<String, List<Piece>>> checkRequestsForPieceItems(Map<String,
                                                                       RequestContext requestContext) {
     var tenantToItem = mapTenantIdsToItemIds(piecesGroupedByPoLine, requestContext);
     return GenericCompositeFuture.all(
-        tenantToItem.entrySet().stream()
-          .map(entry -> {
-          var locationContext = RequestContextUtil.createContextWithNewTenantId(requestContext, entry.getKey());
-          return inventoryItemRequestService.getItemsWithActiveRequests(entry.getValue(), locationContext)
-            .compose(items -> {
-              if (items.isEmpty()) {
-                return Future.succeededFuture();
-              }
-
-              // requestsAction is required to handle outstanding requests
-              if (Objects.isNull(bindPiecesCollection.getRequestsAction())) {
-                logger.warn("checkRequestsForPieceItems:: Found outstanding requests on items with ids: {}", items);
-                throw new HttpException(RestConstants.VALIDATION_ERROR, ErrorCodes.REQUESTS_ACTION_REQUIRED);
-              }
-
-              // When requests are to be transferred check if pieces contain different receivingTenantIds
-              // Transferring requests between tenants is not a requirement for R
-              if (bindPiecesCollection.getRequestsAction().equals(TRANSFER) && !tenantToItem.keySet().isEmpty()) {
-                logger.warn("checkRequestsForPieceItems:: All pieces do not have the same receivingTenantId: {}", tenantToItem.keySet());
-                throw new HttpException(RestConstants.VALIDATION_ERROR, ErrorCodes.PIECES_HAVE_DIFFERENT_RECEIVING_TENANT_IDS);
-              }
-              return Future.succeededFuture();
-            });
-          })
-          .toList())
+      tenantToItem.entrySet().stream()
+        .map(entry -> {
+        var locationContext = RequestContextUtil.createContextWithNewTenantId(requestContext, entry.getKey());
+        return inventoryItemRequestService.getItemsWithActiveRequests(entry.getValue(), locationContext)
+          .compose(items -> validateItemsForRequestTransfer(tenantToItem.keySet(), items, bindPiecesCollection));
+        })
+        .toList())
       .map(f -> piecesGroupedByPoLine);
   }
 
+  private Future<Void> validateItemsForRequestTransfer(Set<String> tenants,
+                                                       List<String> items,
+                                                       BindPiecesCollection bindPiecesCollection) {
+    if (items.isEmpty()) {
+      return Future.succeededFuture();
+    }
+
+    // requestsAction is required to handle outstanding requests
```

Comment:
Renamed all usages

Reply:
Okay I see what you mean. I used outstanding just because the requests were described this way in the story, but essentially it means open requests. Will rename with "open"